### DISCUSSION TITLE
On Reports page pre-set ranges start at year 0001

### DISCUSSION THREAD

**Discussion by jamezul** [firefly-iii/firefly-iii#6797] [Firefly III] [answered]
Hi,

Started my first data import in FireflyIII in 2020 - and i'm using it ever since...

There's one thing that puzzels me: year goes back to year 0001 (my guess is this come form early imports with a worng date format)...
![Screenshot 2023-01-05 at 13 03 23](https://user-images.githubusercontent.com/91373934/210771292-6a797656-ff90-44de-9f6e-9cfe22c6b8fb.png)

Does anyone can help me get rid of 2020 unused years :) ?

Thank you!

**Comment by JC5** [comment, marked as answer]
Find the transactions where the date is wrong. Scroll back on the transactions lists, and you should find it pretty quickly.

**Reply by jamezul** [reply]
Got it!

Thank you so much!
